orttraining/orttraining/test/python/orttraining_on_device_training_tests.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
```

Comment:
Looks like you are running some specific tests applicable to on device training (which are not covered in other CIs). It makes sense, just add some comments explaining the same.